### ISSUE TITLE
Proposal: Eliminate News Redundency

### ISSUE THREAD

**Issue by W-Damian** [lennyhans/7kaa-static-site#7]
Do you think it is necessary to have the news section in the main page? I ask to find out the plan for the site. Also, I would assume the static page would link from 7kfans.com, therefore being redundant displaying it.

**Comment by lennyhans** [comment]
The idea of the page is to be a "hub", so everyone that get interest on the game arrive there and can see what it is, what is going on with it, how to use it and how to contribute (if can apply). That said, the idea is to replace the current 7kfans page as I remember.

**Comment by W-Damian** [comment]
Okay, I didn't know the front page will be changing. I know it is a bit plain tho. I am not a web designer, but I always wanted an area for adding community videos (especially for showing strategy guides) and fan made artwork. I also always dreamed about having the old tournament formula and ranking list back. I can show you form one of the old site not affiliated with Enlight.

**Comment by lennyhans** [comment]
From the forums:

> This is a really simple landing page. I suppose we can use it, but I am looking for some things to be visible on the front page.
> 1. A really short description of the game
> 2. The latest news item. A link for an archive.
> 3. Identify the latest game version, with a quick link to download it.
> 4. A button to take us to the docs (where the converted wiki will live at)
> 5. Stats on the people playing right now (this will be filled in server side -- I'll have a way)

[Forum wiki topic](https://www.7kfans.com/forums/viewtopic.php?f=16&t=1170&start=10#p8891)

The point 2 is why the news on the landing page

**Comment by lennyhans** [comment]
@W-Damian about the community made videos (or promoted players videos ) and the fan artwork that should be added as a new entry point on the menu (like a dropdown on community) then we can add the tournament formula under the play and downloads, we can discuss it in the forums I agree by the way, that will bring more life to the game ecosystem.

**Comment by W-Damian** [comment]
I agree. And personally, I would want a different tab for the documentation vs. general articles.

Keep in mind the scoring formula is copyrighted: "Kingdoms of Kontention scoring system was designed and implemented by Adun (ICQ# 11120986) © 1998, 1999, 2000 Vladimir Atehortúa, iacme L.T.D." 

I used to play S.C.A.R.A.B. on old Mplayer, and it had a ranking system. It was a bit too easy to get up in rank, so I am thinking for 7kaa, we could have our own scoring system if need be to display people's skill and rank (something difficult to achieve), kinda like not playing against a chess master. And I am not sure if you would want to include speed trials as well or scenario scores. I am loki on the forums btw.

**Comment by lennyhans** [comment]
> I agree. And personally, I would want a different tab for the documentation vs. general articles.

Yeah that should be.

> Keep in mind the scoring formula is copyrighted: "Kingdoms of Kontention scoring system was designed and implemented by Adun (ICQ# 11120986) © 1998, 1999, 2000 Vladimir Atehortúa, iacme L.T.D."

Wow, I did not know that, we can reference the copyright, but we should ask if we can use it freely.

> I used to play S.C.A.R.A.B. on old Mplayer, and it had a ranking system. It was a bit too easy to get up in rank, so I am thinking for 7kaa, we could have our own scoring system if need be to display people's skill and rank (something difficult to achieve), kinda like not playing against a chess master. And I am not sure if you would want to include speed trials as well or scenario scores. I am loki on the forums btw.

As far as I know the3dfxdude already has an idea to show the current active games so he has the data of players, we can hook up on that and set the rank system on top of that, of course everyone should agree be shown on the rank system. The speed is a relevant factor, between 7-8. And yeah I deduce that you were loki from the forum, glad to see you back :)

**Comment by W-Damian** [comment]
We may be able to ask Adun if he is @vlaate

**Comment by vlaate** [comment]
Hello folks.

I didn't even know that Enlight had GPL'd 7KAA.

If you need anything from the old Kingdoms of Kontention website (http://iacme.net/7k2/), I hereby release it to the public domain.

**Comment by W-Damian** [comment]
This is good news. Thank you, @vlaate. It's always nice to hear from some of the old players. Amarok has made him way around the forums once or twice.

**Comment by lennyhans** [comment]
That are a really good addition to the possible content of the page!

**Comment by ewbarnes53** [comment]
Please feel free to use any 7K2 material you have from me.  I am surprised to hear that anything I did is still being used.  Say hello to Amarok from me.
-Augustine
    On Thursday, March 25, 2021, 02:50:55 PM CDT, Wednesday Damian Rink ***@***.***> wrote:  
 
 


This is good news. Thank you, @vlaate. It's always nice to hear from some of the old players. Amarok has made him way around the forums once or twice.

In terms of the 7kfans site, we should consider adding 7k2 information. From KoK, some valuable material is
   
   - Rules
   - Scoring System
   - Clan Wars
   - Fryhtan Tournament
   - FAQ

The 7k2 scenarios I mentioned can be downloaded from Kok. Note the credits tho: "this pack contains 58 scenarios created before february 6 2000, by authors like DarkmanX, Real_KD, Bowen, Augustine, Alwee, and others."

There are some pages "Maintained by Augustine," So I am not sure the copyright on them:
   
   - Scenario F.A.Q
   - Campaign Hints
   - Scenario Solutions
   - Scenario Reviews

There is this page which is like the Unit Stats wiki page I was writing based on a couple pages one of the programmers wrote: https://web.archive.org/web/20011226151114/http://iacme.net/7k2/kok/stats/

I wish I could contact Augustine to get some of his material. I am almost tempted to do an @ewbarnes53, but I am not sure if it is the same Augustine.

**Comment by W-Damian** [comment]
There is the old Legions of Heroes at https://web.archive.org/web/20020108165544/http://home.quixnet.net/~kiff/7k2Links1.html

You can download Augustine's Scenario Editor Reference Manual Rev 1.5 there. I used to link to it on the wiki. We could probably host it along with Amarok's guide. I know we plan on hosting the Enlight manual on the static website, so we can include these as well.

**Comment by the3dfxdude** [comment]
To answer the original question. Yes. The front page will no longer be a news reel. It's one reason why the original site is being replaced, because the wordpress structure doesn't have the right feel. Just look at other game sites.

**Comment by lennyhans** [comment]
But there will be a small feed with the recent ones, right?

**Comment by the3dfxdude** [comment]
A very minimal one, if any. The front page news doesn't have to be automated either in the initial version. We can automate things when the need arises.
